template.php

```diff
@@ -133,6 +133,29 @@ function chs_theme_preprocess_islandora_solr_metadata_display(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess().
+ */
+function chs_theme_process_islandora_solr_metadata_display(&$variables) {
+  $solr_fields = $variables['solr_fields'];
+
+  if (isset($solr_fields['mods_accessCondition_useAndReproduction_xlinkhref_ms']) && isset($solr_fields['mods_accessCondition_useAndReproduction_ms'])) {
+    // Take the xlink and node values and create an anchor out of them.
+    $link = reset($solr_fields['mods_accessCondition_useAndReproduction_xlinkhref_ms']['value']);
+    $value = reset($solr_fields['mods_accessCondition_useAndReproduction_ms']['value']);
+
+    $anchor = "<a href='$link'>$value</a>";
```

Comment:
Any particular reason you aren't using [l](https://api.drupal.org/api/drupal/includes%21common.inc/function/l/7.x)?